OpenXmlPowerTools/DocumentBuilder/WmlDocument.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#define TestForUnsupportedDocuments
+#define MergeStylesWithSameNames
+
+using System.Collections.Generic;
+
+namespace OpenXmlPowerTools
+{
+    public partial class WmlDocument : OpenXmlPowerToolsDocument
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=6310620)